docs/using-the-compiler.rst

```diff
@@ -49,19 +51,23 @@ This essentially instructs the compiler to search for anything starting with
 the remapping targets and outside of the directories where explicitly specified source
 files reside, so things like ``import "/etc/passwd";`` only work if you add ``/=/`` as a remapping.
 
-An empty remapping prefix is not allowed.
-
-If there are multiple matches due to remappings, the one with the longest common prefix is selected.
-
-When accessing the filesystem to search for imports, all paths are treated as if they were fully qualified paths.
-This behaviour can be customized by adding the command line option ``--base-path`` with a path to be prepended
-before each filesystem access for imports is performed. Furthermore, the part added via ``--base-path``
-will not appear in the contract metadata.
-
-For security reasons the compiler has restrictions what directories it can access. Paths (and their subdirectories) of source files specified on the commandline and paths defined by remappings are allowed for import statements, but everything else is rejected. Additional paths (and their subdirectories) can be allowed via the ``--allow-paths /sample/path,/another/sample/path`` switch.
+When accessing the filesystem to search for imports, :ref:`relative paths that do not start with ./
+or ../ <relative-imports>` are treated as relative to the directory specified using
+``--base-path`` option (or the current working directory if base path is not specified).
+Furthermore, the part added via ``--base-path`` will not appear in the contract metadata.
 
+For security reasons the compiler has restrictions on what directories it can access.
+Paths of source files (and their subdirectories) specified on the command line and paths defined by
```

Comment:
```suggestion
Directories of source files (and their subdirectories) specified on the command line and paths defined by
```
(Is it directories or paths?)

Reply:
It's directories. Changed to:

```rst
Parent directories of source files specified on the command line and target paths of remappings ...
```

I removed the part about subdirectories because I think it should be pretty intuitive. I'd expect this detail to be mentioned only if it was the opposite.

Reply:
But if you write 'Parent directories', I would interpret this as all (super-)directories leading up to the file, i.e. if you call `solc /a/b/c/d.sol`, then everything inside `/a` is accessible.

Reply:
Right. Why do all the terms have to be so ambiguous!

Anyway, changed `parent directories` to `directories` now.